run.sh

```diff
@@ -279,6 +278,7 @@ if [ -n "$SYNC_SOURCE" ]; then
   fi
   # then fetch a recent mempool.dat,
   curl -f -s -u sync:$SYNC_SECRET -o $DAEMON_DIR/mempool.dat $SYNC_SOURCE || true
```

Comment:
`SYNC_SOURCE` is currently expected to contain the full URL of the mempool file (i.e. `https://blockstream.info/_sync/mempool`). It should be changed to the URL of the `_sync` subdirectory (without the filename), then have it download `$SYNC_SOURCE/mempool` and `$SYNC_SOURCE/fee_estimates` here.

Reply:
thanks didn't catch that, will fix!